package.json

```diff
@@ -6,15 +6,16 @@
   "scripts": {
     "start": "node app",
     "build": "webpack && node app @apostrophecms/asset:build",
-    "dev": "node app @apostrophecms/asset:build && webpack serve & ENV=dev nodemon app"
+    "dev": "nodemon --delay 1000ms -x \"npm run build && ENV=dev nodemon app.js\""
   },
   "nodemonConfig": {
     "verbose": true,
     "watch": [
       "./app.js",
       "./modules/**/*",
       "./lib/**/*.js",
-      "./views/**/*.html"
+      "./views/**/*.html",
+      "./src/**/*"
```

Comment:
We weren't watching the styles for changes through webpack. This is how we're doing it in the demo site.